llvm/test/Transforms/JumpTableToSwitch/basic.ll

```diff
@@ -0,0 +1,94 @@
+; RUN: opt < %s -passes=jump-table-to-switch -verify-dom-info -S | FileCheck %s
+; RUN: opt < %s -passes=jump-table-to-switch -jump-table-to-switch-size-threshold=0 -verify-dom-info -S | FileCheck %s --check-prefix=THRESHOLD-0
+
+@func_array = global [2 x ptr] [ptr @func0, ptr @func1]
```

Comment:
The global should be a constant.
See also `InstCombinerImpl::foldCmpLoadFromIndexedGlobal`.

Reply:
Done